### ISSUE TITLE
[GitDiffView] Open git history via context menu

### ISSUE THREAD

**Issue by jbicker** [theia-ide/theia#1793]
Would be nice to be able to open the git history directly from git file changes and git diff view via a context menu.

**Comment by elaihau** [comment]
Looks like they are now in place ?

![screenshot from 2018-06-21 14-04-28](https://user-images.githubusercontent.com/37082801/41737009-10b9a3f4-755c-11e8-8785-0cfeb747b917.png)

**Comment by jbicker** [comment]
Yes, regarding file navigator you are right. But I've meant that one should be able to open a files history from the GitDiffView too.

**Comment by sgraband** [comment]
`@theia/git` is deprecated; git history is no longer maintained. The VS Code built-in Git extension provides source control features.
